lib/optimize/OccurrenceOrderPlugin.js

```diff
@@ -16,36 +16,40 @@ class OccurrenceOrderPlugin {
 		compiler.plugin("compilation", (compilation) => {
 			compilation.plugin("optimize-module-order", (modules) => {
 				function entryChunks(m) {
-					return m.chunks.map((c) => {
-						const sum = (c.isInitial() ? 1 : 0) + (c.entryModule === m ? 1 : 0);
-						return sum;
-					}).reduce((a, b) => {
-						return a + b;
-					}, 0);
+					var result = 0;
+					var chunks = m.chunks;
+					for(var index = 0; index < chunks.length; index++) {
+						var chunk = chunks[index];
+						result = result + (chunk.isInitial() ? 1 : 0) + (chunk.entryModule === m ? 1 : 0);
+					}
+
+					return result;
 				}
 
 				function occursInEntry(m) {
 					if(typeof m.__OccurenceOrderPlugin_occursInEntry === "number") return m.__OccurenceOrderPlugin_occursInEntry;
-					const result = m.reasons.map((r) => {
-						if(!r.module) return 0;
-						return entryChunks(r.module);
-					}).reduce((a, b) => {
-						return a + b;
-					}, 0) + entryChunks(m);
-					return m.__OccurenceOrderPlugin_occursInEntry = result;
+					var result = 0;
+					var reasons = m.reasons;
+					for(var index = 0; index < reasons.length; index++) {
+						var reasonModule = reasons[index].module;
+						if(!reasonModule) continue;
+						result = result + entryChunks(reasonModule);
+					}
+
+					return m.__OccurenceOrderPlugin_occursInEntry = result + entryChunks(m);
 				}
 
 				function occurs(m) {
 					if(typeof m.__OccurenceOrderPlugin_occurs === "number") return m.__OccurenceOrderPlugin_occurs;
-					const result = m.reasons.map((r) => {
-						if(!r.module) return 0;
-						return r.module.chunks.length;
-					}).reduce((a, b) => {
-						return a + b;
-					}, 0) + m.chunks.length + m.chunks.filter((c) => {
-						c.entryModule === m;
```

Comment:
@sokra There is likely a bug. If correct, then fall 4 test. I edit the code and tests?